packages/sql-indexer/lib/index.js

```diff
@@ -536,6 +540,11 @@ class Indexer {
     toBlock = null,
     skip = null,
   } = {}) {
+    if (lock && type) {
```

Comment:
How about `lock` is a script and `type="empty"` ?

Reply:
The current implementation of notification mechanism doesn't support such subscription yet.

Reply:
OK, I see.